components/run_history_test.go

```diff
@@ -0,0 +1,407 @@
+package components
```

Comment:
`component` directory looks good. can we move the engine and experiment also in under the component directory and this one will be `components/result/run_history_test.go`.

Please note if you do the above change you have to modify in the Makefile for all the tests under engine and experiment as well.

Reply:
@uditgaurav I shifted the engine and experiment tests to the components directory and also made changes in the Makefile, but now if I run the make command it is throwing an error.
Command: `make run-history` 
Error: 
```
stat components/run-history-test.go: no such file or directory
make: *** [Makefile:237: run-history] Error 1
```